shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -1555,19 +1652,48 @@ module MakeImpl<InputSig Lang> {
             PrevStage::callEdgeReturn(call, c, _, _, _, _, _)
           }
 
+          pragma[nomagic]
+          private predicate dataFlowTakenCallEdgeIn0(
+            DataFlowCall call, DataFlowCallable c, ParamNodeEx p, FlowState state, Cc innercc,
```

Comment:
Might as well be `CcCall innercc` for clarity, right?

Reply:
Correct.